File: engine/engine-model/src/main/scala/net/server/channel/packet/command/admin/MonsterHpPacket.scala
package net.server.channel.packet.command.admin

import net.server.MaplePacket

class MonsterHpPacket( private var _mode: Byte,  private var _mobHp: Int) extends BaseAdminCommandPacket(_mode = _mode) {
     def mobHp: Int = _mobHp
}

File: engine/engine-model/src/main/java/net/server/channel/packet/command/admin/MonsterHpPacket.java
package net.server.channel.packet.command.admin;

public class MonsterHpPacket extends BaseAdminCommandPacket {
   private final Integer mobHp;

   public MonsterHpPacket(Byte mode, Integer mobHp) {
      super(mode);
      this.mobHp = mobHp;
   }

   public Integer mobHp() {
      return mobHp;
   }
}
